faucet-service/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Vapor API Template
```

Comment:
We might need a template of our own, or simply remove this until we have one.

Reply:
OK